controllers/users/_preview_scoreboard.htm

```diff
@@ -18,7 +18,7 @@ <h4><?= e(trans('rainlab.user::lang.user.joined')) ?></h4>
             <?= $formModel->created_at->toFormattedDateString() ?>
         </p>
         <p class="description">
-            Status:
+            <?= e(trans('rainlab.user::lang.user.status_text')) ?>:
```

Comment:
This should probably be replaced with a variable language string so that RTL languages can be supported. I.e. `Status: :status`, then call `e(trans('rainlab.user::lang.user.status_text', ['status' => trans('rainlab.user::lang.user.status_guest')]));`